### PR TITLE
create pop up

### PR DESCRIPTION
**Pull request by crestu**
### Changes
Give a short summary of the changes made in this PR. 

so in this pr I added a checkin button to the students page, that renders a partial with a close button

### Link to Issue 
Link Asana ticket here!

https://app.asana.com/0/1206867754300740/1206936335565972

### Comments, Notes, or  Questions
This section is if you want to share something related to the PR with the reviewer. 

### Related Screenshots 
For FE tickets specifcially, please include a screenshot of your work!
Screenshot 2024-04-26 at 12 29 46 PM
Screenshot 2024-04-26 at 12 29 34 PM

### Before you submit for review, Did you...

- [x] Added relevant tests (if applicable)
- [x] Linked Asana ticket to PR
- [x] Added comments, notes, or questions for reviewwer
- [x] Added relevant screenshots
- [x] Ensured ticket title is clear and concise(i.e. 'Donation Form Feature' instead of 'sfg/chels/donation-form'

### CODE REVIEW

File: app/views/students/show.html.erb [hunk 1]
```diff
@@ -1,7 +1,14 @@
 <body class="bg-black overflow-hidden">
+    <div class = "mt-8 flex justify-center">
+      <div class="border border-gray-400 border-1 bg-white px-6 pt-10 pb-8 shadow-xl rounded-lg">
+        <%= link_to "Check In", "#", data: { turbo_frame: "check_in" }, class: "text-blue-800" %>
+      </div>
+    </div>
+  <div class="container flex justify-end h-screen ml-20 mt-[-60px]">
   <div class="container flex justify-end h-screen">
```
Comment (edithemmings) [inline app/views/students/show.html.erb:8]: ^Looks like the container div got repeated here after the merge too... wondering if this is still showing up as expected

**Review by crestu — COMMENTED** [review]
responding to comments

File: app/views/students/show.html.erb (hunk 1 above)
Comment (edithemmings) [reply app/views/students/show.html.erb:8]: You might need to do a `git pull`

**Review by edithemmings — APPROVED** [review]
Looks great! And works on my machine!!

**Review by Muste8 — APPROVED** [review]
NIce work

**Pull request merged**